### ISSUE TITLE
wallaby-node not working in VS Code

### ISSUE THREAD

**Issue by ntrrgc** [wallabyjs/wallaby-typescript-sample#2]
I tried many times selecting the wallaby-node.js file with the _Select configuration file_ but it simply does not work.

VS Code reports it cannot find name `describe` in `greeterNodeSpec.ts` and there is no way to test it.

How are you supposed to test code that uses ES6 modules syntax in TypeScript?

**Comment by ArtemGovorov** [comment]
> VS Code reports it cannot find name describe in greeterNodeSpec.ts

This is not an issue of wallaby.js, it's just VS Code can't find typings. As you can see below I haven't yet started wallaby, but the error is displayed. After selecting the config file, you need to [start wallaby](https://wallabyjs.com/docs/intro/get-started-vscode.html). Selecting wallaby-node config and running wallaby works for me:

![app1](https://cloud.githubusercontent.com/assets/979966/17827341/96f36fbc-66be-11e6-97a2-d66fdeae400c.gif)

I have updated the sample to fix the typings issue by adding the reference comment to the spec. You [may also do it for the whole app](http://stackoverflow.com/questions/37166929/use-typings-in-vscode-without-reference-path-for-each-d-ts). I have also added the ES6 modules syntax.

**Comment by ntrrgc** [comment]
Thanks for the help. Now I got rid of those errors, but still the tests don't work as expected: I get an error if I use `expect()` in `greeterNodeSpec.ts`.

```
expect(true).toBeTruthy();     // expect is not defined
```

**Comment by ArtemGovorov** [comment]
[By default wallaby.js is using `mocha` as a testing framework for node](https://wallabyjs.com/docs/integration/overview.html#supported-testing-frameworks), and `mocha` doesn't have any built-in assertion library, so you have to install `chai` or somethign else. From your assertion code looks like you're using `jasmine`, in this case you need to specify it in the `testFramework` setting:

``` javascript
module.exports = function (w) {

  return {
    ...
    testFramework: 'jasmine'
  };
};
```

+

```
npm install jasmine --save-dev
```

**Comment by ntrrgc** [comment]
But this is a demo project... Shouldn't it come already configured to use
jasmine?

**Comment by ArtemGovorov** [comment]
Fair enough, have added the above mentioned settings to the project.

**Comment by ntrrgc** [comment]
Finally! Thanks.

It would be helpful if test framework errors would be shown though. Until I installed jasmine with npm, the tests did not actually execute and the coverage boxes just kept being white forever. Later I could find there is indeed a "Wallaby.js Console" where the error is logged but I doubt it is obvious for most users just trying out the tool.

![](http://i.imgur.com/rBBk8te.png)
